src/test/scala/no/gdl/bookapi/model/domain/TranslationTest.scala

```diff
@@ -69,6 +69,70 @@ class TranslationTest extends IntegrationSuite with TestEnvironment {
     }
   }
 
+  test("that bookIdsWithLanguageAndLevel returns ids with same language when level is None") {
+    withRollback { implicit  session =>
+      val book1 = addBookDef()
+      val book2 = addBookDef()
+      val book3 = addBookDef()
+
+      val translationForBook1 = addTranslationDef("ext1", "title 1", book1.id.get, "xho", None)
+      val translationForBook2 = addTranslationDef("ext2", "title 2", book2.id.get, "xho", None)
+      val translationForBook3 = addTranslationDef("ext3", "title 3", book3.id.get, "xho", None)
+
+      val ids = Translation.bookIdsWithLanguageAndLevel("xho", None, 10, 1)
+      ids.length should be(3)
+      ids.min should equal (book1.id.get)
+      ids.max should equal (book3.id.get)
```

Comment:
Could these 3 lines be replaced with `ids shouldEqual Seq(book1, book2, book3).map(_.id.get)`? Or does the order matter?

Reply:
And should there be additional books added in this test, with languages `!= "xho"`, to verify that their id's are not included in the results?

Reply:
Regarding order. It is better to use your suggestion. in scala a Seq() is ordered. With the `ids should equal (Seq(book1.id.get, book3.id.get))` We can check both the size and the contents instead of using two statements for this.